platform/ios/src/MGLMapView.h

```diff
@@ -529,12 +529,11 @@ MGL_EXPORT IB_DESIGNABLE
 /**
  A Boolean value that determines whether the updating pitch will also affect the altitude.
  
- When this property is set to `NO`, pitch will work independently from altitude.
- The default value of this property is YES.
- 
- Setting this property to NO will allow animation libraries outside of this SDK to control the
+ Setting this property to `NO` will allow animation libraries outside of this SDK to control the
  map camera without the built-in side effect that pitch influences altitude. This will make the
  results of using outside animation more predictable and easier to control.
+ 
+ The default value of this property is YES.
```

Comment:
We require all types be marked with \` to get rendered properly in our documentation. 
Nit: `YES`